charts/cluster-api/0.1.0/charts/capi-core/templates/deployment.yaml

```diff
@@ -0,0 +1,90 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    cluster.x-k8s.io/provider: cluster-api
+    control-plane: controller-manager
+  name: capi-controller-manager
+  namespace: capi-system
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      cluster.x-k8s.io/provider: cluster-api
+      control-plane: controller-manager
+  template:
+    metadata:
+      labels:
+        cluster.x-k8s.io/provider: cluster-api
+        control-plane: controller-manager
+    spec:
+      containers:
+      - args:
+        - --leader-elect
+        - --diagnostics-address=:8443}
+        - --insecure-diagnostics=false
+        - --feature-gates=MachinePool=false,ClusterResourceSet=false,ClusterTopology=false,RuntimeSDK=false,MachineSetPreflightChecks=false
```

Comment:
Are you likely to want to enable teh optional features in the future?

We enable these by default in turtles:
- MachinePool
- ClusterResourceSet
- ClusterTopology

And we'll likely enable RuntimeSDK soon.

Reply:
Good point - tbh I just copied the configuration from a clusterctl default deployment - I'll add some helm variables and align with the turtles defaults 👍